config/initializers/date_formats.rb

```diff
@@ -5,5 +5,26 @@
 Time::DATE_FORMATS[:month_and_year] = '%B %Y'
 Date::DATE_FORMATS[:month_and_year] = '%B %Y'
 
-Time::DATE_FORMATS[:govuk_date_and_time] = '%e %B %Y at %l:%M%P'
-Time::DATE_FORMATS[:govuk_time] = '%l:%M%P'
+Time::DATE_FORMATS[:govuk_date_and_time] = lambda do |date_and_time|
+  format = if date_and_time == date_and_time.midday
```

Comment:
Comparing dates can always be a little problematic, as it depends on the precision of the objects compared. Are these down to the second or the millisecond etc. More importantly, these will seldom match, whatever the precision. I'd suggest either relaxing the precision or the comparison to allow 12:00:03 to be counted as midday etc.

Reply:
@mnacos good point - amended to compare within the minute of midnight or midday eg. https://github.com/DFE-Digital/apply-for-teacher-training/pull/3606/files#diff-2cd7d9700ea89028447e936ab6d59ebd6187d3ee212db3c1c9791fdd309ff737R9